src/synopsys-action/tools-parameter.ts

```diff
@@ -104,21 +112,38 @@ export class SynopsysToolsParameter {
     return command
   }
 
-  getFormattedCommandForCoverity(userName: string, passWord: string, coverityUrl: string, projectName: string): string {
-    validateCoverityParams(userName, passWord, coverityUrl, projectName)
+  getFormattedCommandForCoverity(userName: string, passWord: string, coverityUrl: string, projectName: string, streamName: string, installDir: string, policyView: string, repositoryName: string, branchName: string): string {
+    validateCoverityParams(userName, passWord, coverityUrl, projectName, streamName)
     const covData: InputData<Coverity> = {
       data: {
         coverity: {
           connect: {
             user: {name: userName, password: passWord},
-            url: coverityUrl,
-            project: {name: projectName},
-            policy: {view: 'SAST'}
-          }
-        }
+            url: coverityUrl
+          },
+          project: {name: projectName},
+          stream: {name: streamName}
+        },
+        project: {}
       }
     }
 
+    if (installDir) {
+      covData.data.coverity.install = {directory: installDir}
```

Comment:
Shall we put validation for installDir for windows environment?

Reply:
done